From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Vladimir Serbinenko <phcoder@gmail.com>
Date: Thu, 31 Dec 2015 18:20:11 +0100
Subject: [PATCH] Disable build-time module check on emu.

On emu some checks can be laxer like check for relocation range. Additionally
module loading in emu is rarely used. So skip this check rather than making
it laxer for all platforms. In ideal we may want to have slightly different
check for emu but for now this is good enough.
---
 grub-core/genmod.sh.in | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/grub-core/genmod.sh.in b/grub-core/genmod.sh.in
index 005cb198504..f74edd55181 100644
--- a/grub-core/genmod.sh.in
+++ b/grub-core/genmod.sh.in
@@ -93,5 +93,7 @@ else
 	    -wd1106 -nu -nd $tmpfile.bin $tmpfile || exit 1
 	rm -f $name.bin
 fi
-./build-grub-module-verifier $tmpfile @target_cpu@
+if test x@platform@ != xemu; then
+    ./build-grub-module-verifier $tmpfile @target_cpu@
+fi
 mv $tmpfile $outfile
